tensorflow_addons/layers/spectral_normalization.py

```diff
@@ -118,11 +117,13 @@ def normalize_weights(self):
             for _ in range(self.power_iterations):
                 v = tf.math.l2_normalize(tf.matmul(u, w, transpose_b=True))
                 u = tf.math.l2_normalize(tf.matmul(v, w))
-
+            u = tf.stop_gradient(u)
```

Comment:
Why test was passing if we didn't stop the gradient? Do we need to improve the test?

Reply:
No, it may improve a little performance.